docs/deep-learning/torch_support.md

```diff
@@ -155,62 +155,93 @@ Concrete ML supports a variety of PyTorch operators that can be used to build fu
-- [`torch.clamp`](https://pytorch.org/docs/stable/generated/torch.clamp.html)
+- [`torch.pow`](https://pytorch.org/docs/stable/generated/torch.pow.html)
+- [`torch.sum`](https://pytorch.org/docs/stable/generated/torch.sum.html)
 - [`torch.mul, torch.Tensor operator *`](https://pytorch.org/docs/stable/generated/torch.mul.html)
 - [`torch.div, torch.Tensor operator /`](https://pytorch.org/docs/stable/generated/torch.div.html)
-- [`torch.nn.identity`](https://pytorch.org/docs/stable/generated/torch.nn.Identity.html)
 - [`torch.nn.BatchNorm2d`](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html)
+- [`torch.nn.BatchNorm3d`](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm3d.html)
+- [`torch.erf, torch.special.erf`](https://pytorch.org/docs/stable/special.html#torch.special.erf)
+- [`torch.nn.functional.pad`](https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html)
 
-#### shape modifying operators
+#### Shape modifying operators
 
 - [`torch.reshape`](https://pytorch.org/docs/stable/generated/torch.reshape.html)
 - [`torch.Tensor.view`](https://pytorch.org/docs/stable/generated/torch.Tensor.view.html#torch.Tensor.view)
 - [`torch.flatten`](https://pytorch.org/docs/stable/generated/torch.flatten.html)
+- [`torch.unsqueeze`](https://pytorch.org/docs/stable/generated/torch.unsqueeze.html)
+- [`torch.squeeze`](https://pytorch.org/docs/stable/generated/torch.squeeze.html)
 - [`torch.transpose`](https://pytorch.org/docs/stable/generated/torch.transpose.html)
+- [`torch.concat, torch.cat`](https://pytorch.org/docs/stable/generated/torch.cat.html)
+- [`torch.nn.Unfold`](https://pytorch.org/docs/stable/generated/torch.nn.Unfold.html)
 
-#### operators that take an encrypted input and unencrypted constants
+#### Tensor operators
+
+- [`torch.Tensor.expand`](https://pytorch.org/docs/stable/generated/torch.Tensor.expand.html)
+- [`torch.Tensor.to`](https://pytorch.org/docs/stable/generated/torch.Tensor.to.html) -- for casting to dtype
+
+#### Multi-variate operators: encrypted input and unencrypted constants
 
-- [`torch.conv2d`, `torch.nn.Conv2D`](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html)
-- [`torch.matmul`](https://pytorch.org/docs/stable/generated/torch.matmul.html)
 - [`torch.nn.Linear`](https://pytorch.org/docs/stable/generated/torch.nn.Linear.html)
+- [`torch.conv1d`, `torch.nn.Conv1D`](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html)
+- [`torch.conv2d`, `torch.nn.Conv2D`](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html)
+- [`torch.nn.AvgPool2d`](https://pytorch.org/docs/stable/generated/torch.nn.AvgPool2d.html)
+- [`torch.nn.MaxPool2d`](https://pytorch.org/docs/stable/generated/torch.nn.MaxPool2d.html)
 
-Concrete ML supports these operators but also the QAT equivalents from Brevitas.
+Concrete ML also supports some of their QAT equivalents from Brevitas.
 
 - `brevitas.nn.QuantLinear`
+- `brevitas.nn.QuantConv1d`
```

Comment:
I have a doubt here : is supporting `torch.conv1d` enough for supporting `brevitas.nn.QuantConv1d` ?

Reply:
I think so yeah

Reply:
I think we support all Brevitas operators that are equivalent to those already supported in Torch. 
So the first sentence seems more appropriate. 

Should we add: to this list: qnn.QuantLinear, qnn.QuantIdentity, qnn.QuantReLU

Reply:
`qnn.QuantLinear` is in this list and `qnn.QuantIdentity` is below in the "Quantizers" sub-section

but for other like activation functions, @andrei-stoian-zama pointed out to me that we should not suggest users to use them as : 
- it's less certain that we indeed do support them
- we most likely never need them after all, as activation functions don't require quantization (done with a TLU), we can just use `torch.relu` 

so I don't think there's much to add here !